pkg/utils/parse_arf_result.go

```diff
@@ -29,15 +31,18 @@ func ParseRemediationsFromArf(scheme *runtime.Scheme, scanName string, namespace
 	// Get the checks that had failed
 	failedRuleResults := filterFailedResults(dom.Root.Query("//TestResult/rule-result"))
 
+	// Get group that contains remediations
+	relevantDom := dom.Root.Query("/report-requests/report-request/content/data-stream-collection/component/Benchmark")
```

Comment:
This is a very nice improvement, but I would suggest to use dom.Root.QueryOne instead

Reply:
apparently the regular query is more performant than QueryOne :/